pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -113,6 +124,65 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, nil
 	}
 
+	if accountClaim.Spec.BYOC {
```

Comment:
BYOC a bool I assume? Don't see it on this PR

Reply:
Yes

Reply:
Its already merged checkout the API types :)